### DISCUSSION TITLE
How can I import / export realm settings on keycloak 18.0.2 (wildfly) without using the graphical interface?

### DISCUSSION THREAD

**Discussion by Baquara** [keycloak/keycloak#16559] [Q&A]
Greetings,

I would like to know how I can import/export realms on Keycloak 18.0.2 Wildfly version.

The guide on:
https://www.keycloak.org/server/importExport
Says I need to pass some parameters to a "kc.sh" file, but as you can see, I cannot find this file in my environment:

![image](https://user-images.githubusercontent.com/9168959/213763310-5f863dbd-26cd-4410-876e-fdb29a5f9bba.png)

**Comment by ssilvert** [comment]
For WildFly, you need to look at the older docs:
https://github.com/keycloak/keycloak-documentation/blob/main/server_admin/topics/export-import.adoc
